src/bin/Interpret.cc

```diff
@@ -257,30 +257,50 @@ void Interpret::interp(ASTNode& n) {
                 break;
             }
             case t_simplify: {
-                sstat status = main_solver->simplifyFormulas();
-                if (status == s_Error)
-                    notify_formatted(true, "Simplify");
+                if (isInitialized()) {
+                    sstat status = main_solver->simplifyFormulas();
+                    if (status == s_Error)
+                        notify_formatted(true, "Simplify");
+                } else {
+                    notify_formatted(true, "Illegal command before set-logic: declare-const");
```

Comment:
```suggestion
                    notify_formatted(true, "Illegal command before set-logic: simplify");
```
Not sure what the exact name is, I did not know we have command for simplification :)

Reply:
Looks like it's `simplify`.  I have no idea what it does :)